Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "zram-generator"
-version = "0.1.2"
+version = "0.2.0-beta.1"
 authors = ["Zbigniew Jędrzejewski-Szmek <zbyszek@in.waw.pl>"]
 license = "MIT"
 description = "Systemd unit generator for zram swap devices"
 homepage = "https://github.com/systemd/zram-generator"
 edition = "2018"
 
 [dependencies]
-anyhow = "1"
+anyhow = "1.0.12"
```

Comment:
Doesn't this hardlock anyhow to 1.0.12? Something like `>=1.0.12` should be more like what you intended

Reply:
No, `1.0.12` means `^1.0.12` that essentially means `>=1.0.12, <2.0.0`

Reply:
Just using `>=1.0.12` is wrong because even 2.x, 3.x and such will match this requirement.